### PR TITLE
Gives Vox a chirp emote and hiss emote

### PR DESCRIPTION
**Pull request by ppi13**
## What Does This PR Do
Give Vox the ability to emote chirps and hiss

## Why It's Good For The Game
Burd sounds are cute, emotes are great.

## Changelog
:cl:
add: Add multiple bird hiss sounds
add: Add multiple bird chirp sounds
add: Add age pitch to hiss emote
add: Add age pitch to chirp emote
/:cl:

### CODE REVIEW

**Comment by MRgaty** [comment]
I dont think chirping or hissing are good sounds for vox.

**Comment by Pixelzilla** [comment]
They sound good to me. I like that there is some variation as well.

**Comment by Pixelzilla** [comment]
> 
> 
> I dont think chirping or hissing are good sounds for vox.

What about the sounds dont you like? What could be improved?

**Comment by iantine** [comment]
I feel that species without sound emotes, such as plasmamen, should be focused on before species that already have sound emotes. However, specific to this PR, the sounds chosen are raptor-like and quite suiting.

**Comment by ppi13** [comment]
If you can think of what sounds plasmamen makes, that can also be added as a future pr, problem is plasmamen are not really something we have an real life equivalent like with vox is to birds and lizards.
But that is more a discussion for another PR

**Comment by Drakeven** [comment]
@Cazdon has an idea for plasmemes, the rattling of bones. Only thing is they need their hands on some wooden wind chimes, which I've been looking for too, to make that BONEY NOISE

**Comment by Drakeven** [comment]
Also, I like the chirp idea- Hissing? Not so much. They already have *quill

**Comment by ppi13** [comment]
both the chirps and hisses comes from vultures

**Comment by taukausanake** [comment]
> plasmamen are not really something we have an real life equivalent

I always imagined them to sound kind of like Endermen but I think the Drask already sound like that kind of. Maybe give them some Darth Vader "skrrrrk" sound because the suit

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -209,16 +217,87 @@
 			playsound(loc, 'sound/voice/dionatalk1.ogg', 50, 0) //Credit https://www.youtube.com/watch?v=ufnvlRjsOTI [0:13 - 0:16]
 			m_type = 2
 
+		if("chirp", "chirps")
+			var/M = handle_emote_param(param)
+			message = "<B>[src]</B> chirps[M ? " at [M]" : ""]."
+			switch(rand(1,10))
+				if(1)
+					playsound(loc, 'sound/effects/voxchirp01.ogg', 50, 1, frequency = get_age_pitch()) //Credit to AlexTriceratops123  (https://www.youtube.com/watch?v=uA5yGyB_z5U) for the sound.
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: This code is really gross.

Just make it something like `playsound(loc, 'sound/effects/voxchirp[rand(1,10)].ogg', 50, 1, frequency = get_age_pitch())`

And make adjustments to the naming structure.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -209,16 +217,87 @@
...
+					m_type = 2
+				else
+					message = "<B>[src]</B> makes a weak hissing noise."
+					m_type = 2
+
+			if(isvox(src))
+				if(!muzzled)
+					message = "<B>[src]</B> hisses[M ? " at [M]" : ""]."
+					switch(rand(1,10))
+						if(1)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: Same

**Comment by ppi13** [comment]
I believe all to be in order now.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -97,6 +97,8 @@
 		if("hiss", "hisses")
 			if(isunathi(src)) //Only Unathi can hiss.
 				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
+			if(isvox(src)) //Only Vox can hiss.
```
Comment (farie82) [inline code/modules/mob/living/carbon/human/emote.dm:100]: Best to just include this in the check on line 98. Also the comments are conflicting!!!

File: code/modules/mob/living/carbon/human/emote.dm [hunk 4]
```diff
@@ -112,12 +114,22 @@
...
+			else								//Everyone else fails, skip the emote attempt
+				return
+
 		if("scream", "screams")
 			on_CD = handle_emote_CD(50) //longer cooldown
 		if("fart", "farts", "flip", "flips", "snap", "snaps")
 			on_CD = handle_emote_CD()				//proc located in code\modules\mob\emote.dm
 		if("cough", "coughs", "slap", "slaps", "highfive")
 			on_CD = handle_emote_CD()
+		if("gasp", "gasps")
```
Comment (farie82) [inline code/modules/mob/living/carbon/human/emote.dm:129]: Not sure why this is shown as an added line of code... might be due to the outdated branch

File: code/modules/mob/living/carbon/human/emote.dm [hunk 5]
```diff
@@ -137,115 +149,127 @@
 		if("howl", "howls")
 			var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
 			message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
-			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 0, 10)
+			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 1, 10, frequency = get_age_pitch())
```
Comment (farie82) [inline code/modules/mob/living/carbon/human/emote.dm:152]: Same here. Something might have gone wrong with your merge with the master. Last one i'll comment on

**Comment by Fox-McCloud** [comment]
No response since March. Again, nothing wrong with this PR, but it's super stale, even with request changes.

**Pull request closed without merging**